src/components/builds/BuildCard.tsx

```diff
@@ -78,16 +85,23 @@ export default function BuildCard(props: Props) {
   );
 
   let classes = useStyles();
-  let theme = useTheme();
   const navigate = useNavigate();
+  const isDrawerOpen = useRecoilValue(cirrusOpenDrawerState);
 
   const pageWidth = usePageWidth();
 
-  let isMdScreenWidth = pageWidth >= theme.breakpoints.values.md;
-
+  // For default Chip component values
   const themeOptions = useRecoilValue(muiThemeOptions);
   const muiTheme = useMemo(() => createTheme(themeOptions), [themeOptions]);
 
+  // For screens with open Drawer
+  const muiThemeOptionsCustom = useRecoilValue(muiCustomBreakpoints);
+  const muiCustomBreakpointsTheme = createTheme(muiThemeOptionsCustom);
```

Comment:
What do you think of modifying `themeOptions.breakpoints` based on `isDrawerOpen` before passing to `createTheme`.

Just substract the drawer width from all the `breakpoints` values.